readme.md

```diff
@@ -0,0 +1,766 @@
+# Readme
+
+## Meta
+
+- Last updated: 2016-07-01
+- Release version: 1.6.1
```

Comment:
Aside the fact, that the informations are outdated (two years old), I tend to agree in general because it's a source of unnecessary maintenance effort.